packages/trader/src/App/Containers/ResetPasswordModal/reset-password-modal.jsx

```diff
@@ -0,0 +1,180 @@
+import classNames        from 'classnames';
+import {
+    Input,
+    Button,
+    Dialog }             from 'deriv-components';
+import {
+    Formik,
+    Form }               from 'formik';
+import PropTypes         from 'prop-types';
+import React             from 'react';
+import Localize          from 'App/Components/Elements/localize.jsx';
+import { localize }      from 'App/i18n';
+import { connect }       from 'Stores/connect';
+import { validPassword } from 'Utils/Validator/declarative-validation-rules';
+import { redirectToLogin } from '_common/base/login';
+
+const resetInitialValues = { password: '' };
+
+class ResetPassword extends React.Component {
```

Comment:
Don't we need the password strength meter here too? it's in shared and can look at `deriv-password.jsx` for implementation

Reply:
design dont use password meter, but definitely good idea to use password meter instead